clue.js

```diff
@@ -0,0 +1,181 @@
+  color: 'yellow',
+  description: 'He is good at weapons',
+  age: 55,
+  image: 'assets/mustard.png',
+  occupation: 'Hunter'
+}
+const mrsPeacock = {
+  firstName: 'Lena',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She has a lot of money',
+  age: 50,
+  image: 'assets/peacock.png',
+  occupation: 'Rich Mama'
+}
+const professorPlum = {
+  firstName: 'Lennart',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'He is a genius',
+  age: 35,
+  image: 'assets/plum.png',
+  occupation: 'Genius'
+}
+const missScarlet = {
+  firstName: 'Scarlet',
+  lastName: 'Johansson',
+  color: 'red',
+  description: 'She is beautiful',
+  age: 30,
+  image: 'assets/scarlet.png',
+  occupation: 'Beauty'
+}
+const mrsWhite = {
+  firstName: 'Edna',
+  lastName: 'White',
+  color: 'white',
+  description: 'She knows everything',
+  age: 60,
+  image: 'assets/white.png',
+  occupation: 'Know it all'
+}
+
+// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+
+const rope = {
+  name: 'rope',
+  weight: 5,
+  image: 'assets/rope.png'
+}
+const gun = {
+  name: 'gun',
+  weight: 40,
+  image: 'assets/gun.png'
+}
+const knife = {
+  name: 'knife',
+  weight: 20,
+  image: 'assets/knife.png'
+}
+const wrench = {
+  name: 'wrench',
+  weight: 50,
+  image: 'assets/wrench.png'
+}
+const candlestick = {
+  name: 'candlestick',
+  weight: 30,
+  image: 'assets/candle.png'
+}
+const pipe = {
+  name: 'pipe',
+  weight: 10,
+  image: 'assets/pipe.png'
+}
+
+
+// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+
+
+// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+
+const suspects = [
+  mrGreen,
+  mrMustard,
+  mrsPeacock,
+  professorPlum,
+  missScarlet,
+  mrsWhite
+]
+
+const weapon = [
+  rope,
+  gun,
+  knife,
+  wrench,
+  candlestick,
+  pipe
+]
+
+const room = [
+  'hall',
+  'lounge',
+  'study',
+  'library',
+  'kitchen',
+  'ballroom'
+]
+
+// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
+// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+const randomSelector = array => {
+  return array[Math.floor(Math.random() * array.length)]
+}
+
+// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
+// With a killer, a weapon and a room.
+// The values will be set later.
+let crime = {
+ killer: [],
+ weapon: [],
+ room: ''
+}
+
+// This function will be invoked when you click on the killer card.
+const pickKiller = () => {
+  // This will randomly select a killer from the suspects. And add that to the mystery object.
+  mystery.killer = randomSelector(suspects)
+
+  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
+  document.getElementById('killerCard').style.background = mystery.killer.color;
+  document.getElementById('killerCard').addEventListener('click', pickKiller)
```

Comment:
Nice that you used the addEventListener here to call the function when clicked on, instead of writing it in the HMTL element. I think you might need to move this to outside the function so it can be called? Maybe the teacher can clarify this as I'm not sure.

Reply:
I checked whole file and you are right Claire - line 146 is 100% irrelevant and, being honest, it won't work. You cannot assign function to an event inside that funtion.
Still, you've done assignment in line 179, so everything work.
This line however (146) needs to be deleted, same goes for other functions in your code